target/linux/ramips/image/mt7628.mk

```diff
@@ -26,6 +30,17 @@ define Device/miwifi-nano
 endef
 TARGET_DEVICES += miwifi-nano
 
+define Device/tl-wr840n-v4
+  DTS := TL-WR840NV4
+  IMAGE_SIZE := 7808k
+  DEVICE_TITLE := TP-Link TL-WR840N v4
+  DEVICE_PACKAGES :=
+  KERNEL := $(KERNEL_DTB) | prepend-to 448 | uImage lzma
```

Comment:
I'm afraid it's not possible.
`KERNEL_DTB = kernel-bin | patch-dtb | lzma`
Calling _pad-extra_ before making KERNEL_DTB causes, that all leading zeroes crated by pad-extra are subject of _lzma_ a will be comprimed:(
I can't find better/simpler solution.